### PR TITLE
SMHE-2067 SetClockConfiguration config

### PR DESCRIPTION
**Pull request by kroesctrl**
SMHE-2067 SetClockConfiguration config

### CODE REVIEW

**Comment by kroesctrl** [comment]
https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-smhe-cucumber-build/235/

**Comment by kroesctrl** [comment]
https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-smhe-cucumber-build/236/

**Comment by kroesctrl** [comment]
https://jenkins.fdp.osgp.cloud/view/SMHE-cucumber/job/gxf-smhe-cucumber-build/238/

File: osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/GetActualPowerQualityCommandExecutorTest.java [hunk 1]
```diff
@@ -405,7 +423,7 @@ private List<AttributeAddress> getAttributeAddresses(final List<CosemObject> obj
 
   private CosemObject createCosemObject(final String tag, final int classId, final String obis) {
     return new CosemObject(
-        tag, "descr", classId, 0, obis, "group", null, List.of(), Map.of(), List.of());
+        tag, "descr", classId, 0, obis, "group", null, List.of(), Map.of(), new ArrayList<>());
```
Comment (stefanermens) [inline osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/GetActualPowerQualityCommandExecutorTest.java:426]: Why is `List.of()` changed to `new ArrayList<>()`?

File: osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json [hunk 2]
```diff
@@ -98,6 +98,43 @@
...
+          "description": "daylights_savings_end",
+          "datatype": "octet-string",
+          "valuetype": "SET_BY_CLIENT",
+          "access": "RW"
+        },
+        {
+          "id": 7,
+          "description": "daylights_savings_deviation",
+          "datatype": "integer",
+          "valuetype": "SET_BY_CLIENT",
```
Comment (stefanermens) [inline osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json]: This attribute can't be written by the client. It's fixed in the profile. Same applies for other protocol versions.

File: osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json [hunk 3]
```diff
@@ -98,6 +98,43 @@
           "valuetype": "DYNAMIC",
           "value": "OK",
           "access": "R"
+        },
+        {
+          "id": 5,
+          "description": "daylights_savings_begin",
```
Comment (stefanermens) [inline osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json]: Correct naming is `daylight_savings_begin`, as is specified in the DLMS blue book. The spelling `daylights_savings` in (D)SMR seems to be a mistake.

**Review by stefanermens — DISMISSED** [review]

File: osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/GetActualPowerQualityCommandExecutorTest.java (hunk 1 above)
Comment (kroesctrl) [reply osgp/protocol-adapter-dlms/osgp-protocol-adapter-dlms/src/test/java/org/opensmartgridplatform/adapter/protocol/dlms/domain/commands/misc/GetActualPowerQualityCommandExecutorTest.java:426]: In the test i add some attributes to the object, this is not allowed with a List.

File: osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json (hunk 2 above)
Comment (kroesctrl) [reply osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json]: valuetype changed to FIXED_IN_PROFILE

File: osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json (hunk 3 above)
Comment (kroesctrl) [reply osgp/protocol-adapter-dlms/osgp-dlms/src/main/resources/dlmsprofiles/dlmsprofile-dsmr22.json]: Indeed i used the (D)SMR, there it is "daylight_savings_". I will change it config to "daylight_saving_"
Scherm­afbeelding 2024-04-29 om 08 24 52

**Review by natasja-n — APPROVED** [review]

**Pull request merged**